### ISSUE TITLE
rikaichamp fails to build for chrome

### ISSUE THREAD

**Issue by halcow** [birtles/rikaichamp#92]
[npm-debug.log](https://github.com/birtles/rikaichamp/files/3068655/npm-debug.log)

**Comment by birtles** [comment]
Hmm, that's odd. `npm run build:chrome` works fine for me on Ubuntu. Did you run `npm install` first?

**Comment by halcow** [comment]
Yes, at least I believe I did, either way that doesn't seem to help it, but I noticed I do get some warnings when running `npm install`, I'm unfamiliar with npm so I'm unsure about their significance but I'll post them anyway just in case.

>npm WARN optional Skipping failed optional dependency /addons-linter/fsevents:
>npm WARN notsup Not compatible with your operating system or architecture: fsevents@2.0.1
>npm WARN optional Skipping failed optional dependency /chokidar/fsevents:
>npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.2.7
>npm WARN optional Skipping failed optional dependency /sane/fsevents:
>npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.2.7
>npm WARN karma-webpack@3.0.5 requires a peer of webpack@^2.0.0 || ^3.0.0 but none was installed.
>npm WARN ts-jest@24.0.2 requires a peer of jest@>=24 <25 but none was installed.
>npm WARN ajv-keywords@2.1.1 requires a peer of ajv@^5.0.0 but none was installed.
>npm WARN rikaichamp@0.0.32 No license field.

And here's another [npm-debug.log](https://github.com/birtles/rikaichamp/files/3077139/npm-debug.log) in case I did forget to run `npm install` before.
By the way, building for firefox works fine.
